plugins/versionpress/admin/deactivate.php

```diff
@@ -16,10 +16,12 @@
  * @param string $type See `submit_button()`'s $type parameter
  * @param string $cssClass Optional CSS class to be used for the button
  */
-function _vp_button($label, $action, $type = "delete", $cssClass = "") {
-    echo "<form action='" . esc_url(admin_url('admin-post.php')) . "' method='post' class='" . esc_attr($cssClass) . "'>";
+function _vp_button($label, $action, $type = "delete", $cssClass = "")
+{
+    echo "<form action='" . esc_url(admin_url('admin-post.php')) .
+        "' method='post' class='" . esc_attr($cssClass) . "'>";
```

Comment:
These are the situations where I feel like the standard is more terrorizing us than helping. 123 chars line length is fine.

I've checked [PSR-2](http://www.php-fig.org/psr/psr-2/) to confirm that 120 is indeed the hard limit but it's not:

> There MUST NOT be a hard limit on line length; the soft limit MUST be 120 characters; lines SHOULD be 80 characters or less.

From [1. Overview](http://www.php-fig.org/psr/psr-2/#1-overview) and [2.3 Lines](http://www.php-fig.org/psr/psr-2/#2-3-lines). This will make long HTML lines and similar stuff much easier.